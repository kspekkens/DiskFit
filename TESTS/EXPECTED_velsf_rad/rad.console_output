 DiskFit (v1.2.2)  Copyright (C) 2015, 2017, Jerry Sellwood and Kristine Spekkens

 This program comes with ABSOLUTELY NO WARRANTY.  It is free software
 and you are welcome to redistribute it under certain conditions.
 See <http://www.gnu.org/licenses/> for details.

 Enter input parameter file name

 Read in input parameters from: EXAMPLE/VELS/FITS/velsf_rad.inp                                                                     

Input disk eps: (1-b/a)  0.45
Input disk PA (N->E; deg) =  45.00
Input x-center (data units) =   130.50
Input y-center (data units) =   126.50
Input disk systemic velocity (km/s) =   110.00

 Fitting for:
 disk PA
 disk ellipticity
 disk x-center, y-center
 radial flows
 disk systemic velocity
 --> Other attributes fixed to input values

 Velocity field components extracted at radii:
    3.0    9.0   15.0   21.0   27.0   33.0   39.0   45.0   51.0   57.0
   63.0   69.0   75.0   81.0   87.0   93.0   99.0  105.0  111.0  117.0
  123.0  129.0  135.0  141.0  147.0

 Fitting for m=0 radial flow in addition to rotation velocities
 Radial flows set to 0 in rings beyond    111.000000    

 Read in measured velocities and uncertainties from:
 EXAMPLE/VELS/FITS/vels.fits


 Number of datapoints read in:        1224
 Number of datapoints to be used in fit:        1224
   diskpa:   diskel:     xcen:     ycen:
   45.0000  0.450000  130.5000  126.5000
starting iteration    1 best reduced chisq so far: 1.351E+02
   diskpa:   diskel:     xcen:     ycen:
   45.0000  0.450000  130.5000  126.5000
starting iteration    2 best reduced chisq so far: 2.213E+01
   diskpa:   diskel:     xcen:     ycen:
   40.0463  0.504207  126.7115  127.9621
starting iteration    3 best reduced chisq so far: 9.051E+00
   diskpa:   diskel:     xcen:     ycen:
   40.1700  0.492253  128.4647  128.5463
starting iteration    4 best reduced chisq so far: 8.852E+00
   diskpa:   diskel:     xcen:     ycen:
   40.0029  0.500249  128.4954  128.5519
starting iteration    5 best reduced chisq so far: 8.852E+00
   diskpa:   diskel:     xcen:     ycen:
   40.0045  0.500464  128.4963  128.5500

 Done minimization
 Total number of iterations:            5
 Minimum chi^2 found:    8.85155582    

 Wrote:  EXAMPLE/VELS/FITS/OUT/rad.mod.fits
 Wrote:  EXAMPLE/VELS/FITS/OUT/rad.res.fits
 Wrote out: EXAMPLE/VELS/FITS/OUT/rad.out
