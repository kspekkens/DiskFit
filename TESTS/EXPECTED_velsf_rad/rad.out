Minimization output, vels
-------------------------
Input files:
EXAMPLE/VELS/FITS/velsf_rad.inp
EXAMPLE/VELS/FITS/vels.fits   pixscale  1.00 arcsec/pix


FITS region:       x-low:      1   y-low:      1 x-range:    255 y-range:    255
FITS sampling:    regrad:  12.00   regpa:  45.00  regeps:   0.45 istpout:      6

Output model and (data-model) residuals files:
EXAMPLE/VELS/FITS/OUT/rad.mod
EXAMPLE/VELS/FITS/OUT/rad.res

Disk toggles:         PA: T     eps: T  center: T non-axi: F    phib: F
vels toggles:    interp0: T  radial: T    Vsys: T    warp: F      rw: F    welm: F   wphim: F

Input values
------------
disk PA, phi_d^prime (deg):           45.00
disk eps:                              0.45
x,y center (data units):             130.50  126.50


Vsys (km/s):                         110.00
Delta_ISM (km/s):                      1.00





No seeing correction applied
No model component smoothing applied
No uncertainties estimated
Non-axisymmetries fixed to 0 outside range:    3.00  111.00


Best fitting values
-------------------
disk PA, phi_d^prime (deg):           40.00 +/-  0.00
disk eps:                              0.50 +/-  0.00
disk incl (deg):                      60.03 +/-  0.00
x,y center (data units):             128.50 +/-  0.00,  128.55 +/-  0.00

Vsys (km/s):                         100.04 +/-  0.00









Minimization Details
---------------------
# points Dn used in fit:               1224
# iterations in minimization:             5
Minimum chi^2 found:               8.851556
Degrees of freedom in fit:             1175

Fitted velocity components                  (radii in data units, velocities in km/s):
       r        npts          Vt         eVt          Vr         eVr        Vm,t       eVm,t        Vm,r       eVm,r
--------------------------------------------------------------------------------------------------------------------
      3.00     56.59       63.22        0.00       -4.81        0.00        0.00        0.00        0.00        0.00
      9.00    134.76      116.05        0.00      -12.80        0.00        0.00        0.00        0.00        0.00
     15.00     59.88      131.27        0.00      -18.99        0.00        0.00        0.00        0.00        0.00
     21.00     11.76      139.84        0.00      -22.21        0.00        0.00        0.00        0.00        0.00
     27.00     13.82      143.97        0.00      -20.82        0.00        0.00        0.00        0.00        0.00
     33.00     16.53      145.69        0.00      -27.29        0.00        0.00        0.00        0.00        0.00
     39.00     21.66      150.95        0.00      -28.11        0.00        0.00        0.00        0.00        0.00
     45.00     23.07      150.17        0.00      -25.23        0.00        0.00        0.00        0.00        0.00
     51.00     26.23      154.16        0.00      -26.73        0.00        0.00        0.00        0.00        0.00
     57.00     29.85      159.65        0.00      -24.19        0.00        0.00        0.00        0.00        0.00
     63.00     34.11      167.52        0.00      -16.75        0.00        0.00        0.00        0.00        0.00
     69.00     35.02      175.28        0.00      -12.39        0.00        0.00        0.00        0.00        0.00
     75.00     38.75      180.94        0.00       -5.78        0.00        0.00        0.00        0.00        0.00
     81.00     43.75      185.65        0.00       -2.95        0.00        0.00        0.00        0.00        0.00
     87.00     45.11      189.85        0.00       -3.46        0.00        0.00        0.00        0.00        0.00
     93.00     47.66      190.58        0.00       -2.33        0.00        0.00        0.00        0.00        0.00
     99.00     52.64      190.76        0.00        0.60        0.00        0.00        0.00        0.00        0.00
    105.00     55.32      192.30        0.00       -0.89        0.00        0.00        0.00        0.00        0.00
    111.00     57.72      192.27        0.00       -0.03        0.00        0.00        0.00        0.00        0.00
    117.00     60.11      192.62        0.00        0.00        0.00        0.00        0.00        0.00        0.00
    123.00     65.91      192.70        0.00        0.00        0.00        0.00        0.00        0.00        0.00
    129.00     67.12      193.67        0.00        0.00        0.00        0.00        0.00        0.00        0.00
    135.00     69.66      193.05        0.00        0.00        0.00        0.00        0.00        0.00        0.00
    141.00     74.11      194.68        0.00        0.00        0.00        0.00        0.00        0.00        0.00
    147.00     78.26      194.62        0.00        0.00        0.00        0.00        0.00        0.00        0.00
